SetupCrossTenantRelationshipForTargetTenant.ps1

```diff
@@ -282,6 +320,8 @@ function Create-KeyVaultAndGenerateCertificate([string]$targetTenant, `
                                                $retrieveCertPrivateKey, `
                                                [string]$auditStorageAcntRG, `
                                                [string]$auditStorageAcntName, `
+                                               [string]$auditStorageAcntLocation, `

```

Comment:
can we use the same location as kv location?

Reply:
I think they are different, I need to use `westus` as storage account's location, and `West US` as kv location..